### ISSUE TITLE
Statusmsgs aren't distinguishable if they're '/me's

### ISSUE THREAD

**Issue by 0xallie** [weechat/weechat#124]
If someone sends a `/me` to for example `+#test`, it will show as a regular `/me` in `#test`.

```
* +tester - this is a normal message
* +tester - this is a message to +#test
```

**Comment by flashcode** [comment]
Hi,

Could you please give me the commands to reproduce the issue?

And I know… almost 9 years to answer, but at least I'm trying to fix such old bugs right now!

**Comment by flashcode** [comment]
Duplicate of #1577, fixed in version 4.0.0.

When sending these commands:

```
/ctcp @#test action is testing (for ops)
/ctcp #test action is testing (normal)
```

The output is now:

```
-- | Action -> @#test: FlashCode is testing (for ops)
 * | FlashCode is testing (normal)
```
